server/test/services/applicant/ApplicantServiceFastForwardEnabledTest.java

```diff
@@ -3163,6 +3164,8 @@ public void maybeEligibleProgramsForApplicant_doesNotIncludeIneligibleSubmittedA
                     .setEligibilityIsGating(false)
                     .setAcls(new ProgramAcls())
                     .setCategories(ImmutableList.of())
+                    .setApplicationSteps(
```

Comment:
For some reason these tests aren't failing without a short description, which is also required. I'm not sure why they require application steps but not description.

Reply:
Hm, I can't seem to find an explanation.